cADFS.psm1

```diff
@@ -940,31 +940,33 @@ class cADFSNode {
     [System.Boolean] Test() {
         # Assume compliance by default
         $Compliant = $true;
-
+        $SyncProperties = $null
 
         Write-Verbose -Message 'Testing for presence of Active Directory Federation Services (ADFS) farm.';
 
         try {
-            $Properties = Get-AdfsProperties -ErrorAction Stop;
+            $SyncProperties = Get-AdfsSyncProperties -ErrorAction Stop;
         }
         catch {
-            $Compliant = $false;
```

Comment:
Rather than an empty catch block I'd remove the whole try/catch and remove the ErrorAction Stop